js/shared_controls.js

```diff
@@ -1,35 +1,28 @@
 if (!Array.prototype.indexOf) {
-	Array.prototype.indexOf = function(searchElement, fromIndex) {
-	var k;
-
-	if (this == null) {
-		throw new TypeError('"this" equals null or n is undefined');
-	}
-
-	var O = Object(this);
-
-	var len = O.length >>> 0;
-
-	if (len === 0) {
-		return -1;
-	}
-
-	var n = +fromIndex || 0;
-
-	if (Math.abs(n) === Infinity) {
-		n = 0;
-	}
-	if (n >= len) {
-		return -1;
-	}
-
-	k = Math.max(n >= 0 ? n : len - Math.abs(n), 0);
-	while (k < len) {
-		if (k in O && O[k] === searchElement) {
-	    	return k;
-	  	}
-	  	k++;
-	}
+	this.indexOf = function (searchElement, fromIndex) {
```

Comment:
This should still be `Array.prototype.indexOf =`.

You can tell eslint to ignore this if it's still an issue.